Tests/OpenCombineTests/FoundationTests/PropertyListDecoderTests.swift

```diff
@@ -5,6 +5,10 @@
 //  Created by Sergej Jaskiewicz on 10.12.2019.
 //
 
+// PropertyListEncoder and PropertyListDecoder are unavailable in
+// swift-corelibs-foundation prior to Swift 5.1.
+#if canImport(Darwin) || swift(>=5.1) && os(Linux) // TEST_DISCOVERY_CONDITION
+
```

Comment:
```suggestion
// PropertyListEncoder and PropertyListDecoder are unavailable in
// swift-corelibs-foundation prior to Swift 5.1.
#if canImport(Darwin) || swift(>=5.1) && !WASI // TEST_DISCOVERY_CONDITION

```

Please also update the comment after `#endif.`